### DISCUSSION TITLE
@xstate/inspect window focus

### DISCUSSION THREAD

**Discussion by tjdam** [statelyai/xstate#2694] [Q&A] [answered]
Hi all!

I have this liiiittle (really little) pet peeve with inspect that is the inspector window (or tab) gets focused when states change,  so I have to either go back and forth tabs on each transition or keep the inspect tab detached from the running project window, which makes it a bit clumsy to go back and forth.

Is there any config to change this behavior? Is it something I have to solve on the browser settings? If so, how?

Thank you for the great job, can't live without it anymore <3

**Comment by davidkpiano** [comment, marked as answer]
Unfortunately no, but we will be publishing guides on how to get stately.ai/viz working in embedded mode so it will be easier to use as an `<iframe>` instead. The normal browser behavior is to focus on the newly opened window, and there doesn't seem to be a way to change that behavior.
